src/handlers/showcase.ts

```diff
@@ -13,27 +13,39 @@ export default async function handleShowcaseMessage(
       config.FORTIES_SHOWCASE
     )) as TextChannel;
 
-    const url = msg.attachments.first()?.proxyURL as string
-    console.log(
-      `40s channel posted showcase: ${msg.author.username} ${url}`
-    );
+    const url = msg.attachments.first()?.proxyURL as string;
+
+    if (!url) {
+      console.log('No showcase image!');
+      return;
```

Comment:
```suggestion
      await msg.reply('something went wrong. We'll take a look, but in the meantime please ensure an image is attached and try again.');
      return;
```
I'm curious if this block will ever be reachable, but in the event that is what is going on w/ discord in the one-offs where we don't see an image displayed in the embed, then I think it would be beneficial to have some user feedback.

Reply:
Done! https://github.com/40-percent/40percent-bot/pull/43/files#diff-51bab71b77ed0e0ecc5250300b23a979dce7832d6ec08026484cb8f26183e9bbR26